urlparse4/cgurl.pyx

```diff
@@ -245,3 +247,18 @@ def urljoin(base, url, allow_fragments=True):
         return joined_url
 
     return stdlib_urljoin(base, url, allow_fragments=allow_fragments)
+#
```

Comment:
did you mean to uncomment it? or add in anotheer PR?

Reply:
I will work on implementing this function now, Konstantin!